rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](./interop_2022.md), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will manage this process and make decisions based on consensus, see [governance](#governance).
+
+There will be a dashboard at https://wpt.fyi/interop-2023.
+
+## Details
+
+The Interop 2022 [repo](https://github.com/web-platform-tests/interop-2022) and [team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will be renamed to omit the year and used for this effort as well.
+
+### Timeline
+
+During 2022:
+
+- September 15 to October 15: Proposals period.
+- October 16 to October 30: Proposal discussion/refinement. (New proposals no longer accepted.)
+- November 1 to November 30: Proposal selection. (The team must reach consensus no later than November 30.)
+- December 1 to December 16:
+  - Start review of tests. Write needed tests.
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop) will manage the overall Interop 2023 effort. The team will be formed by renaming the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition.
+
+The Interop 2023 effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+### Proposals
+
+During the proposals period (September 15 to October 15), proposals are submitted as issues in the [Interop repo](https://github.com/web-platform-tests/interop). A proposal should be as specific as reasonably possible, but multiple proposals may be grouped later in the selection process.
```

Comment:
OK, so this is likely overkill, but if the proposals were PRs instead, this would be pretty clear: the actual PR is the proposal. If you want to work with the other parties to make changes then you have that conversation in the PR issue. If you want to fork the proposal then you create a new PR.

In practice I don't want to have to judge between N slight variations on the ~same proposal, so I definitely don't want everyone to be encouraged to fork rather than collaborate.

Reply:
@jgraham on balance, do you think we should do proposals as PRs or not?

Or we ask that the initial comment is updated to reflect the complete proposal at each point?

Either way it's going to be a bit annoying in trivial cases where there's no actual ambiguity, and useful in case where the discussion has made things murky.

Reply:
Well I'm interested to see what others think about proposal-as-PR.

Technically, the more I think about it the more advantages it has. We can merge the actual PRs so that the repo has a list of all the accepted proposals and various metadata e.g. the wpt.fyi labels used. We can have year-dated directories to keep a history of the project without having to dig through GH issues. We could in theory have an up-to-date list of tests in each proposal and verify that matches the labels on wpt.fyi (or write tooling that will auto-sync the labels). Although all these things are possible with issues, the fact that you have to go through the GH API to access the data does make things harder.

But it does have some additional overhead (perhaps less now that you can simply use github.dev to edit inline). So if that's a blocker for people I'm happy to withdraw this suggestion.

Reply:
OK let's take proposals-as-PR as a serious suggestion (or a proposal-proposal, if you will).

Reply:
I'd like us to have an in-repo record of what Interop 2023 is defined as, and doing that as PRs makes sense. My only concern would be raising the bar too much for proposals by people who don't have all of the details ready. Perhaps we accept issues, but at some point there needs to be a PR that formalizes the proposal?

Reply:
To some extent you can use whatever method you want to workshop a proposal, but you are correct that it could be worth calling out that we are happy for people to use the repo issue tracker to help scope out ideas for proposals.

Reply:
I've added this:

> To make proposal scope unambiguous, accepted proposals should be fully documented in the repository. To this end, and to make proposal review easier, the team may require that proposals are submitted as pull requests. Whether and how discussions, issues and pull requests are used in the proposals process is left to the team to decide before the proposal period begins.